### PR TITLE
Renamed roles

### LINKED ISSUE #21: Rename Roles
Roles will be renamed according to this proposal from @Flaxscrip  approved by @startupsocieties : https://github.com/ulex-opensource/Lexinomicon/issues/20#issuecomment-501497631

### PR DESCRIPTION
**Pull request by macterra**
Resolves #21. Preview >> https://macterra.github.io/Lexinomicon/Roles/

### CODE REVIEW

**Review by joestartupsocieties — APPROVED** [review]

File: Roles/Director/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Director
+
+1. The Director [role](..) is responsible for representing the customer/end-user/stakeholders for a given project or task.
```
Comment (macterra) [inline Roles/Director/README.md:3]: Should we reference the ICG Managing Director here or defer?

Comment (joestartupsocieties) [reply Roles/Director/README.md:3]: If you think it's necessary, say that the Director is held by the ICG managing director. And that the treasurer is held by the Ulex Advisory board.

Comment (macterra) [reply Roles/Director/README.md:3]: OK done

**Review by Flaxscrip — APPROVED** [review]
Awesome work @macterra! Thank you for implementing all this in the rules. Now we're all Legislators!

**Pull request merged**